src/database/schema/ChatSchema.ts

```diff
@@ -280,6 +318,7 @@ class ChatSchema implements BaseDbSchema {
     userId: string,
     channelId: string,
     message: string,
+    attachments: any,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider defining a type for `attachments` instead of using `any`.

```diff
-     attachments: any,
+     attachments: AttachmentInterface[],
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->